index.js

```diff
@@ -1,5 +1,13 @@
 var querystring = require('querystring')
-var xhr = require('xhr')
+var xhr;
+
+// check if running in browser or as server
+try {
+  xhr =  require('xhr');
```

Comment:
Could you replace the try/catch blog with a simple `if (xhr.open)`? Makes the whole thing smaller and more readable.